pkg/omf/cli/omf.theme.fish

```diff
@@ -1,6 +1,64 @@
-function omf.theme
-  if not test -e $OMF_CONFIG/themes/$argv[1]
-    omf.install --theme $argv[1]
+function omf.theme -a target_theme
+  set -l current_theme (cat $OMF_CONFIG/theme)
+
+  test "$target_theme" = "$current_theme"; and begin; echo Already at; return 0; end
+
+  if not contains -- "$target_theme" (omf.list_installed_themes)
+    if not omf.install --theme "$target_theme"
```

Comment:
I believe this is not part of the theme method. The CLI should know that we need to install the theme and then call omf theme.